setup.py

```diff
@@ -31,7 +31,7 @@ def read(*rnames):
 
 setup(
     name="django-multiselectfield",
-    version="0.1.12",
+    version="0.1.13",
```

Comment:
There may be additional commits/changes merged after this change, which would make this version bump look out of place in the Git history. Please remove it. Us maintainers will bump it when we're making a release. 😃

Reply:
Sorry about that I'll remove this change from the PR

Reply:
Don't worry about it, I think I fixed this in #148. If you get a minute, please review that.

Reply:
I reviewed the PR and you fixed the issue! We can just close my PR since the 148 does it better